search.ipynb

```diff
@@ -144,15 +144,3534 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Iteration: 1; Guess of 1.00e+03 produced a keff of 1.08504 +/- 0.00169\n",
-      "Iteration: 2; Guess of 2.50e+03 produced a keff of 0.95243 +/- 0.00158\n",
-      "Iteration: 3; Guess of 1.75e+03 produced a keff of 1.01269 +/- 0.00163\n",
-      "Iteration: 4; Guess of 2.12e+03 produced a keff of 0.98165 +/- 0.00155\n",
-      "Iteration: 5; Guess of 1.94e+03 produced a keff of 0.99773 +/- 0.00158\n",
-      "Iteration: 6; Guess of 1.84e+03 produced a keff of 1.00872 +/- 0.00170\n",
-      "Iteration: 7; Guess of 1.89e+03 produced a keff of 1.00462 +/- 0.00154\n",
-      "Iteration: 8; Guess of 1.91e+03 produced a keff of 1.00202 +/- 0.00154\n",
-      "Iteration: 9; Guess of 1.93e+03 produced a keff of 0.99816 +/- 0.00155\n",
+      "                                %%%%%%%%%%%%%%%\n",
```

Comment:
Can you update this so that we don't get the OpenMC output from every single iteration? It would be good if we just got the Iteration summary (like it was before)

Reply:
Done! Added the appropriate entry for `run_args`.